src/azure-cli/azure/cli/command_modules/acr/custom.py

```diff
@@ -62,6 +62,9 @@ def acr_create(cmd,
     if sku not in get_managed_sku(cmd):
         raise CLIError("Classic SKU is no longer supported. Please select a managed SKU.")
 
+    if registry_name.isupper():
+        raise CLIError("argument error: Connected registry name must be only lowercase.")
```

Comment:
remove "connected"

Reply:
Curious, should we block this on RP? I think we allow today so someone can still use REST api and create ACR with uppercase names.

Reply:
I was asked to do it in the front end to avoid the user experiencing a bug that can be found. It's a faster fix I guess?